pages/windows/reg-flags.md

```diff
@@ -0,0 +1,19 @@
+# reg flags
+
+> Display or set flags on registry keys.
+
+- Display current flags for a specific key:
+
+`reg flags {{key_name}} query`
```

Comment:
What is a query ?

Reply:
This is the command for displaying currently active flags for a key.

Query is just the keyword.